src/components/widget/MasterWidget.js

```diff
@@ -167,7 +156,7 @@ class MasterWidget extends Component {
    * @param {*} property
    * @param {*} val
    */
-  handleChange = (property, val) => {
+  handleChange = (property, val, cachedVal) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['cachedVal' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/metasfresh/metasfresh-webui-frontend/pullRequest?prid=4578381)